### DISCUSSION TITLE
Zooming into the relevant area of a JFreeChart.

### DISCUSSION THREAD

**Discussion by Hadiiiiiiiiiiiiii** [jfree/jfreechart#334] [Q&A]
Hey,
i would like to know if there is a way to only zoom in on the relevant area of a graph, for example if my graph begins at 6500 and ends at 6800 is there a way to only show the area from 6400 to 6900? or something similar to that while keeping the axis.

**Comment by trashgod** [comment]
This [example](https://stackoverflow.com/a/67230096/230513) uses [`setRange()`](https://www.jfree.org/jfreechart/javadoc/org/jfree/chart/axis/ValueAxis.html#setRange(double,double)) to adjust the corresponding axis so that it flanks the desired range.
